### PR TITLE
Add hack for :placeholder-shown

### LINKED ISSUE #1311: Support for :placeholder-shown
Autoprefixer does not currently "translate" :placeholder-shown

(I'm NOT talking about [::placeholder](https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder) which is a different thing and already supported)

- https://developer.mozilla.org/en-US/docs/Web/CSS/:placeholder-shown#Browser_compatibility
- https://caniuse.com/#feat=css-placeholder-shown
- https://dev.to/thesheebs/comment/aog1

Example 1:

```CSS
input:placeholder-shown {
  border-color: silver;
}
```

```CSS
/* IE >= 10 */
input:-ms-input-placeholder {
  border-color: silver;
}

/* Firefox 4 -> 50 included */
/* After performing some tests, this does not work */
input:-moz-placeholder {
  border-color: silver;
}
```

Example 2:

```CSS
input:not(:placeholder-shown) {
  background-color: red;
}
```

```CSS
/* IE >= 10 */
input:not(:-ms-input-placeholder) {
  background-color: red;
}

/* Firefox 4 -> 50 included */
/* After performing some tests, this does not work */
input:not(:-moz-placeholder) {
  background-color: red;
}
```

### PR DESCRIPTION
**Pull request by tkrotoff**
Fixes #1311

Please review carefully, this is my first PR here.

### CODE REVIEW

File: test/autoprefixer.test.js [hunk 1]
```diff
@@ -54,6 +54,9 @@ let ffgradienter = autoprefixer({
 let selectorer = autoprefixer({
   overrideBrowserslist: ['Chrome 25', 'Firefox > 17', 'IE 10', 'Edge 12']
 })
+let placeholderShowner = autoprefixer({
+  overrideBrowserslist: ['IE >= 10']
```
Comment (tkrotoff) [inline test/autoprefixer.test.js:58]: I'm not sure about that

File: data/prefixes.js [hunk 2]
```diff
@@ -362,6 +362,15 @@ f(require('caniuse-lite/data/features/css-placeholder'), browsers => {
   })
 })
 
+// Placeholder-shown selector
+f(require('caniuse-lite/data/features/css-placeholder-shown'), browsers => {
+  prefix([':placeholder-shown'], {
+    selector: true,
+    feature: 'css-placeholder-shown',
+    browsers
```
Comment (tkrotoff) [inline data/prefixes.js:370]: css-placeholder uses `browsers: browsers.concat(['ie 10 old', 'ie 11 old', 'firefox 18 old'])` here. I don't understand what is 'ie 10 old' vs 'ie 10'.

**Comment by ai** [comment]
LGTM. I will try to review it deeply and release on weekend.

File: data/prefixes.js (hunk 2 above)
Comment (ai) [reply data/prefixes.js:370]: On my machine it is:

```
    [
      'firefox 4',  'firefox 5',  'firefox 6',
      'firefox 7',  'firefox 8',  'firefox 9',
      'firefox 10', 'firefox 11', 'firefox 12',
      'firefox 13', 'firefox 14', 'firefox 15',
      'firefox 16', 'firefox 17', 'firefox 18',
      'firefox 19', 'firefox 20', 'firefox 21',
      'firefox 22', 'firefox 23', 'firefox 24',
      'firefox 25', 'firefox 26', 'firefox 27',
      'firefox 28', 'firefox 29', 'firefox 30',
      'firefox 31', 'firefox 32', 'firefox 33',
      'firefox 34', 'firefox 35', 'firefox 36',
      'firefox 37', 'firefox 38', 'firefox 39',
      'firefox 40', 'firefox 41', 'firefox 42',
      'firefox 43', 'firefox 44', 'firefox 45',
      'firefox 46', 'firefox 47', 'firefox 48',
      'firefox 49', 'firefox 50', 'ie 10',
      'ie 11',      'kaios 2.5'
    ]
```

Comment (ai) [reply data/prefixes.js:370]: We use ` old` postfix in flexbox and some other prefixes with 2 syntaxes for the same prefix

**Comment by ai** [comment]
Thanks. Released at 9.8.

**Pull request merged**